guacamole-common-js/src/main/webapp/modules/Tunnel.js

```diff
@@ -214,6 +214,10 @@ Guacamole.HTTPTunnel = function(tunnelURL) {
 
         // Mark as closed
         tunnel.state = Guacamole.Tunnel.State.CLOSED;
+        
+        sendingMessages = false;
```

Comment:
Please add a comment describing what this is doing at a high-level. "Reset output message buffer", for example, would be acceptable.